bin/gem_downloader

```diff
@@ -20,7 +20,22 @@ Dir.chdir(gp.target_dir) {
   gp.target_files.each { |n, v, an|
     path = sprintf(file_format, index, n, v)
     loop {
-      `curl -o #{path} -L http://rubygems.org/downloads/#{n}-#{v}.gem`
+      if v.include?('gem-fetch-')
```

Comment:
Want to document the special meaning of this prefix somewhere?

Reply:
I've added a comment for it. PTAL.